enh(network::hp::standard::snmp) : added vrrp-status CTOR-712 (#4637)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-hp-standard-snmp.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

The following monitoring connectors will be installed when you install the **HP Standard Network SNMP** connector through the

> **Configuration > Monitoring Connector Manager** menu:

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu:
```

</TabItem>
<TabItem value="Uptime" label="Uptime">

| Name                  | Unit  |
|:----------------------|:------|
| system.uptime.seconds | s     |

> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.

</TabItem>
<TabItem value="Vrrp-Status" label="Vrrp-Status">

| Name   | Unit  |
|:-------|:------|
| status | N/A   |

> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.

</TabItem>
</Tabs>

## Prerequisites

### SNMP Configuration

The SNMP service must be configured and activated on the host. 

> Please refer to the official documentation from the constructor/editor.

Reviewer: standard prerequisites?

> The SNMP service must be configured and activated on the host. 
> Please refer to the official documentation from the constructor/editor.

Author: ```suggestion
The SNMP service must be configured and activated on the host. 
Please refer to the official documentation. 
Your resource may require a list of addresses authorized to query it to be set up. 
Please ensure that the addresses of the Centreon pollers are included in this list.
```

### Pack

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the

> **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
**Configuration > Connectors > Monitoring Connectors** menu.
```

</TabItem>
</Tabs>

2. Whatever the license type (*online* or *offline*), install the **HP Standard Network SNMP** connector through

> the **Configuration > Monitoring Connector Manager** menu.

Reviewer: ```suggestion
the **Configuration > Connectors > Monitoring Connectors** menu.
```
